www/src/pages/blog/astro-018.astro

```diff
@@ -0,0 +1,112 @@
+---
```

Comment:
is there a reason that this couldn't be a markdown file, similar to `posts/introducing-astro.md`? It would be nice to stay consistent, and reuse the `BlogPost` layout which seems to be doing most of what's been copied here

Reply:
There isn't a `posts/introducing-astro.md`. There's a posts `posts/introducing-astro.astro` which appears to not be linked to `blog/introducing-astro.astro` is what's linked.

Reply:
Would it be possible to delete the /posts/introducing-astro.astro and add a vercel redirect to the /blog path?

Reply:
Ah shoot, you're right I was looking at `examples/blog` and not `www/blog`. Nevermind!

Reply:
It would be good to move this into a markdown file though so that the two could share this layout logic, but that's less pressing.

Reply:
This one probably could be, but the other is using `<Note />` and a couple of other components so I think it can't be until we get `.mdc` files.

Reply:
yup, probably blocked by our "components in markdown" work

Reply:
> Would it be possible to delete the /posts/introducing-astro.astro and add a vercel redirect to the /blog path?

I made a PR for this here: https://github.com/snowpackjs/astro/pull/876 - I might be missing something but I assume we don't want duplicated files for the blog?